indexer/services/l1/bridge/bridge.go

```diff
@@ -25,59 +26,62 @@ type Bridge interface {
 type implConfig struct {
 	name string
 	impl string
-	addr common.Address
+	addr string
 }
 
-var addrs map[string]common.Address
+var defaultBridgeCfgs = []*implConfig{
+	{"Standard", "StandardBridge", predeploys.DevL1StandardBridge},
+	{"ETH", "ETHBridge", predeploys.DevL1StandardBridge},
```

Comment:
These are L1 predeploys for the devnet - you can't use them against a real network. Is there any way to set these at runtime/via config?

Reply:
Typically these should be configured _once_ after deployment on the network and mostly never change after that. Not sure if it's worth it to make it configurable. Instead I've updated to map chainID to the bridge addresses like we're doing with custom bridges below.